ow_faults_injection/src/FaultInjector.cpp

```diff
@@ -0,0 +1,199 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include "ow_faults_injection/FaultInjector.h"
+#include <algorithm>
+
+using namespace ow_lander;
+
+using std::string;
+// static constexpr float64 SET_FLAG = 1.0;
+
+FaultInjector::FaultInjector(ros::NodeHandle& node_handle)
+{ 
+  //  arm pub and subs
+  const char* joint_states_str = "joint_states";
+  m_joint_state_sub = node_handle.subscribe(string("/_original/") + joint_states_str, 10,
+    &FaultInjector::jointStateCb, this);
+  m_joint_state_remapped_pub = node_handle.advertise<sensor_msgs::JointState>(joint_states_str, 10);
+  m_joint_state_flags_pub = node_handle.advertise<ow_faults_injection::JointStatesFlag>(string("/flags/") + joint_states_str, 10);
```

Comment:
Line too long (exceeds coding guideline max of 120 chars).  Other lines in this function could be shortened for better readability.  One thing that would help is renaming `node_handle` to `nh`.  In this context, the abbreviation would actually enhance readability, I think.